### ISSUE TITLE
loss_cls

### ISSUE THREAD

**Issue by x-x110** [chensnathan/YOLOF#22]
during training, you get the index of box which iou more than 0.7 and less 0.10. however, using pred_class_logits[valid_idxs] to compute loss，to set more than 0.7 sanple are negative .why?
i think that boxes greater than 0.7 should be classified accurately. So, may be set these box to get training?

**Comment by chensnathan** [comment]
Sorry for the late reply.

You can refer to [this answer](https://github.com/megvii-model/YOLOF/issues/5#issuecomment-802882241) for details.

**Comment by x-x110** [comment]
thank you for your reply and i meet a new question. During training, compared with ~0.5 of baseline, my total loss get 0.43 , 0.18 for box and 0.25 for cls, but the map get 35.9 at iter 30000. i am provided that overfit not happen. Since, the map is improve gradually at val.
this scene has happens more than once.

**Comment by chensnathan** [comment]
What did you modify in this round?

**Comment by x-x110** [comment]
sample matching method. box loss is mean and cls loss is loss_sum/num(positive)

**Comment by x-x110** [comment]
so the loss reflect the  average loss，it doesn't matter how you match box.

**Comment by chensnathan** [comment]
This is your own experiment. You may need to debug and analyze it by yourself.

**Comment by x-x110** [comment]
OK, thanks for you reply

**Comment by x-x110** [comment]
OK, thanks for you reply
